buildSrc/src/main/kotlin/org/sonarsource/kotlin/buildsrc/tasks/CreateRuleStubsTask.kt

```diff
@@ -177,3 +193,37 @@ private object Templates {
         
     """.trimIndent()
 }
+
+private object KotlinGradleTemplates: Templates {
+    override val checksDir = Path.of("sonar-kotlin-gradle", "src", "main", "java", "org", "sonarsource", "kotlin", "gradle", "checks")
+    override val testsDir = Path.of("sonar-kotlin-gradle", "src", "test", "java", "org", "sonarsource", "kotlin", "gradle", "checks")
+    override val samplesDir = Path.of("sonar-kotlin-gradle", "src", "test", "samples", "non-compiling")
+    override val checkListFile =
+        Path.of(KOTLIN_GRADLE_BASEDIR, "src", "main", "java", "org", "sonarsource", "kotlin", "gradle", "KotlinGradleCheckList.kt")
+
+    override fun generateCheckClass(ruleKey: String, checkClassName: String, messageLine: String?) = LICENSE_HEADER + """
+        package org.sonarsource.kotlin.gradle.checks
+        
+        import org.sonar.check.Rule
+        import org.sonarsource.kotlin.api.checks.AbstractCheck
+        
+        ${messageLine?.let { "        \n        $it\n" } ?: ""}
```

Comment:
Remove the empty line to avoid a superfluous empty line
```suggestion
        ${messageLine?.let { "        \n        $it\n" } ?: ""}
```

Reply:
Done.